itemloaders/__init__.py

```diff
@@ -26,6 +26,69 @@ def unbound_method(method):
 
 
 class ItemLoader:
+    """
+    Return a new Item Loader for populating the given item. If no item is
+    given, one is instantiated automatically using the class in
+    :attr:`default_item_class`.
+
+    When instantiated with a :param ``selector`` parameter the :class:`ItemLoader` class
+    provides convenient mechanisms for extracting data from web pages
+    using parsel_ selectors.
+
+    :param item: The item instance to populate using subsequent calls to
+        :meth:`~ItemLoader.add_xpath`, :meth:`~ItemLoader.add_css`,
+        or :meth:`~ItemLoader.add_value`.
+    :type item: :class:`dict` object
+
+    :param selector: The selector to extract data from, when using the
+        :meth:`add_xpath` (resp. :meth:`add_css`) or :meth:`replace_xpath`
+        (resp. :meth:`replace_css`) method.
+    :type selector: :class:`~scrapy.selector.Selector` object
+
+    The item, selector and the remaining keyword arguments are
+    assigned to the Loader context (accessible through the :attr:`context` attribute).
+
+    .. attribute:: item
+
+        The item object being parsed by this Item Loader.
+        This is mostly used as a property so when attempting to override this
+        value, you may want to check out :attr:`default_item_class` first.
+
+    .. attribute:: context
+
+        The currently active :ref:`Context <loaders-context>` of this Item Loader.
+        Refer to <loaders-context> for more information about the Loader Context.
+
+    .. attribute:: default_item_class
+
+        An Item class (or factory), used to instantiate items when not given in
+        the ``__init__`` method.
+
+    .. attribute:: default_input_processor
+
+        The default input processor to use for those fields which don't specify
+        one.
+
+    .. attribute:: default_output_processor
+
+        The default output processor to use for those fields which don't specify
+        one.
+
+    .. attribute:: default_selector_class
+
+        The class used to construct the :attr:`selector` of this
+        :class:`ItemLoader`, if only a response is given in the ``__init__`` method.
```

Comment:
there is no response support in itemloaders

Reply:
is default_selector_class used in itemloaders, or should it be moved to scrapy.loader?

Reply:
Indeed, it's not necessary here.
Removed